README.md

```diff
@@ -0,0 +1,67 @@
+# BackTREK
+
+## Introduction
+
+For this project we'll be returning to the TREK API, using Backbone to build an application that can handle data in complex ways.
+
+## Learning Goals
+
+- Use Backbone and jQuery to work with a complex API
+- Manage application data using Backbone models and collections
+- Build an attractive, robust, and feature-rich user interface
+
+## Wave 1
+
+In wave 1, you'll use our new functionality (Underscore templates, Backbone models and collections) to achieve feature parity with the original TREK project, including some of the optionals:
+
+- A user can click a link to see a list of trips
+- A user can click on a trip to see details of that trip
+- A user can create a new trip
+- A user can reserve a spot on a trip
+  - Question: is Backbone API integration useful here? How would this work? What other options are there?
+- The code you write should obey the following rules:
+  - Use Backbone's event workflow. Code that responds to DOM events should be separate from code that updates the DOM.
+  - Any dynamic elements on the page shall be rendered using Underscore templates
+  - Any errors encountered while interacting with the API shall be politely reported to the user
+
+You do not need to implement sorting or filtering yet.
+
+## Wave 2
```

Comment:
There are two waves to the project but I don't see the waves being assigned separately. Are we providing both waves at the beginning of the project time?

Reply:
When I put this together, I was thinking about this project being assigned on Monday, where they don't yet have the tools to do sorting or filtering. Now that we're doing MixCD on Monday and this is on Tuesday, I'm not sure the separation is necessary.

Maybe I'll break it up into:
- Displaying trips
- POSTing data (new trip, reservation)
- Managing the collection

I do think we'll assign all three at once, and students can pace themselves.